api/v1/linstorsatellite_types.go

```diff
@@ -75,6 +75,34 @@ type LinstorSatelliteStatus struct {
 	// +listType=map
 	// +listMapKey=type
 	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+
+	// The number of volumes in the LINSTOR Cluster.
```

Comment:
On this satellite/node rather than "in the cluster". Same for other fields below.